src/components/DeFi/components/AssetInput.tsx

```diff
@@ -188,13 +188,11 @@ export const AssetInput: React.FC<AssetInputProps> = ({
         </Stack>
       )}
       {handleAccountIdChange && assetId && (
-        <Stack direction='row' py={2} px={4} justifyContent='space-between' alignItems='center'>
-          <AccountDropdown
-            assetId={assetId}
-            onChange={handleAccountIdChange}
-            buttonProps={{ variant: 'ghost', width: 'full', padding: 0 }}
-          />
-        </Stack>
+        <AccountDropdown
```

Comment:
`AccountDropdown` can't be in a Stack or it causes some conflicts with the Popper library used by Chakra.

Reply:
@0xApotheosis this may be fixed by @reallybeard yesterday with the `zIndex` here if that's what you were experiencing? https://github.com/shapeshift/web/blob/develop/src/components/AccountDropdown/AccountDropdown.tsx#L281

Reply:
Yah the dropdown has a pretty high zIndex now, so should help stay on top of everything else.

Reply:
Nah, it's unrelated to zIndex, it's this guy:

<img width="881" alt="Screen Shot 2022-09-15 at 5 22 04 am" src="https://user-images.githubusercontent.com/97164662/190243926-c54fb322-0ceb-4db1-8c75-a48f5bd278d0.png">